src/gui/application.cpp

```diff
@@ -175,11 +177,11 @@ Application::Application(int &argc, char **argv) :
     // Also check immediatly
     QTimer::singleShot( 0, this, SLOT( slotCheckConnection() ));
 
-    if( cfg.skipUpdateCheck() ) {
```

Comment:
the check is now in Application::slotStartUpdateDetector(). Even if we skip the update stack *now*, we might not in 10 hours, thats why we need the timer even if skip now.